File: 2. Loop Problems/Loop problems in python/p12.py
# Python program to calculate power of a number using a for loop


n = int(input("Enter the number (base): "))
power = int(input("Enter the power (exponent): "))


result = 1


for i in range(1, power + 1):
    result *= n  

print(f"Result = {result}")

File: 2. Loop Problems/Loop Problems in c/p12.c
// Write a C program to find power of a number using for loop

#include<stdio.h>
#include<math.h>

int main() {
    int n,power;
    long long result = 1;

    printf("Enter number and power: ");
    scanf("%d%d",&n,&power);

    for (int i = 1; i<= power ; i++) {
        result = result * n;
    }
    printf("result = %lld\n",result);
}

